### PR TITLE
Travel Website

### PR DESCRIPTION
**Pull request by EscHaleem**
Mobile Travel Website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        </li>
+        <li> 
+            <p>Guide</p>
+        </li>
+        <li> 
+            <p>Contact Us</p>
+        </li>
+    </ul>
+</nav>
+</footer>
```
Comment (Devsalmasaeed) [inline index.html]: #  Navigation should be at the header not footer

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+/* tablet */
+
+
+
+/* desktops */
+.nav ul{
+    list-style-type: none;
+    margin: 0px;
+}
```
Comment (Devsalmasaeed) [inline styles.css:872]: Hi @EscHaleem,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
-  **Good use of HTML Semantic** 👍
-  **Professional PR title** 👍
-  **Professional PR description** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

#  Adjust the width of mirror lake card

- The Mirror Lake should have the same width with the remaining cards
Step 1 screenshot

<br/>

#  Remove the hover effect on all the cards

- Remove the hover effect on each card

Step 1 screenshot

- This is how your cards should be display

Step 2 screenshot

<br/>

#  Make sure your ul li are display like the figma design

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @EscHaleem ,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- Please make the sections below to be same as the figma design
- 
image
- allign this section to be central just like in the figma desgin
-
image

- Also add readme file

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Xfarouq — APPROVED** [review]
Hi @EscHaleem 

Your project is complete! There is nothing else to say other than... it's time to merge it!!!
Congratulations! 🎉

<div style="text-align:center;">
    Remote Collaboration
</div>

### Highlights
- ✅ **Professional Readme**
- ✅ **Good use of Gitflow**
- ✅ **UI looks like the Figma design**
- ✅ **Good use of HTML semantics**
- ✅ **Professional commit messages**
- ✅ **Professional PR title**
- ✅ **Professional PR description**

### Optional Suggestions
- Add the link to your website in your readme
- Rotate the blue blurry image in the header section to be upright

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — COMMENTED** [review]
> Hi @EscHaleem
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it!!! Congratulations! 🎉
> 
> Remote Collaboration
> ### Highlights
> * ✅ **Professional Readme**
> * ✅ **Good use of Gitflow**
> * ✅ **UI looks like the Figma design**
> * ✅ **Good use of HTML semantics**
> * ✅ **Professional commit messages**

> 
> ### Optional Suggestions
>  * Add  Professional PR title and description for your desktop view
> 
> Cheers and Happy coding! 👏👏👏
> 
> _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification._

**Pull request merged**